src/UrqlClient.re

```diff
@@ -81,6 +79,9 @@ module ClientTypes = {
     partialOpUrl: string,
     [@bs.optional] [@bs.as "meta"]
     partialOpDebugMeta: operationDebugMeta,
+    // TODO: how to add this to useQuery...
```

Comment:
I was trying to find out how to only allow this in `useQuery`

Reply:
Gotcha. I don't think it's possible to restrict this property to _only_ be used by a `partialOperationContext` used by `useQuery` _and not_ `useMutation`. I think that's ok – it's something we can flag in docs.

To add this as an option that the `useQuery` hook can supply as part of the config rather than via a `partialOperationContext`, you can do the following:

1. Add `[@bs.optional] pollInterval: int` here: https://github.com/FormidableLabs/reason-urql/blob/master/src/hooks/UrqlUseQuery.re#L3-L12
2. Ensure you add an argument for it here, `pollInterval=?`: https://github.com/FormidableLabs/reason-urql/blob/master/src/hooks/UrqlUseQuery.re#L89
3. Make sure it gets forwarded along here: https://github.com/FormidableLabs/reason-urql/blob/master/src/hooks/UrqlUseQuery.re#L97 as `pollInterval?`

And then we'll want to follow the same steps for the `Query` component in `UrqlQuery.re`.